core/quantity/src/Parser.ts

```diff
@@ -325,30 +352,51 @@ export class Parser {
     // common case where single value and single label are supplied
     if (tokens.length === 2) {
       if (tokens[0].isNumber && tokens[1].isString) {
-        const unit = await unitsProvider.findUnit(tokens[1].value as string, defaultUnit ? defaultUnit.unitFamily : undefined);
-        return new Quantity(unit, tokens[0].value as number);
+        const unit = await this.lookupUnitByLabel(tokens[1].value as string, format, unitsProvider);

```

Comment:
There is a try/catch around the `lookupUnitByLabel` above, would the same workflow be appropriate here?  I'm reading through this and I don't see any handling for if the unit isn't found based on the token provided. Is it just expecting that it will throw in that case?

Reply:
The try/catch must have been from an early implementation... there is nothing being thrown or expected to be thrown. I will remove the old try/catch.

Reply:
removed unneeded try/catch.